refactoring/refactoring.php

```diff
@@ -2,50 +2,49 @@
 
 public function post_confim() {
     $id = Input::get('service_id');
-    $servicio = Service::find($id);
-    //dd($servicio)
-    if ($servicio != NULL) {
-        if ($servicio->status_id == '6') {
-            return Response::json(array('error' => '2'));
```

Comment:
Avoid `array('error' => '2')` better create a class with all constants with the different error, is more easy to read and more easy to maintain the code

```php
array('error' => Error::Two)
```